### ISSUE TITLE
rqt can no longer find any plugins

### ISSUE THREAD

**Issue by sven-herrmann** [ros-visualization/rqt#240]
Since Python 3.9 the previously deprecated function getiterator on ElementTree has been removed.
Thus rqt is no longer able to find any plugins.
The output of rqt is:
```
provider "<class 'rqt_gui.rospkg_plugin_provider.RospkgPluginProvider'>":
Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3.9/site-packages/qt_gui/composite_plugin_provider.py", line 57, in discover
    plugin_descriptors = plugin_provider.discover(discovery_data)
  File "/opt/ros/noetic/lib/python3.9/site-packages/rqt_gui/ros_plugin_provider.py", line 68, in discover
    plugin_descriptors += self._parse_plugin_xml(package_name, plugin_xml)
  File "/opt/ros/noetic/lib/python3.9/site-packages/rqt_gui/ros_plugin_provider.py", line 121, in _parse_plugin_xml
    for library_el in root.getiterator('library'):
AttributeError: 'ElementTree' object has no attribute 'getiterator'

 provider "<class 'rqt_gui.rospkg_plugin_provider.RospkgPluginProvider'>":
Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3.9/site-packages/qt_gui/composite_plugin_provider.py", line 57, in discover
    plugin_descriptors = plugin_provider.discover(discovery_data)
  File "/opt/ros/noetic/lib/python3.9/site-packages/rqt_gui/ros_plugin_provider.py", line 68, in discover
    plugin_descriptors += self._parse_plugin_xml(package_name, plugin_xml)
  File "/opt/ros/noetic/lib/python3.9/site-packages/rqt_gui/ros_plugin_provider.py", line 121, in _parse_plugin_xml
    for library_el in root.getiterator('library'):
AttributeError: 'ElementTree' object has no attribute 'getiterator'

 provider "<class 'qt_gui.recursive_plugin_provider.RecursivePluginProvider'>":
Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3.9/site-packages/qt_gui/composite_plugin_provider.py", line 57, in discover
    plugin_descriptors = plugin_provider.discover(discovery_data)
  File "/opt/ros/noetic/lib/python3.9/site-packages/qt_gui/recursive_plugin_provider.py", line 53, in discover
    plugin_descriptors = self._plugin_provider.discover(discovery_data)
  File "/opt/ros/noetic/lib/python3.9/site-packages/rqt_gui/ros_plugin_provider.py", line 68, in discover
    plugin_descriptors += self._parse_plugin_xml(package_name, plugin_xml)
  File "/opt/ros/noetic/lib/python3.9/site-packages/rqt_gui/ros_plugin_provider.py", line 121, in _parse_plugin_xml
    for library_el in root.getiterator('library'):
AttributeError: 'ElementTree' object has no attribute 'getiterator'

 provider "<class 'qt_gui.recursive_plugin_provider.RecursivePluginProvider'>":
Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3.9/site-packages/qt_gui/composite_plugin_provider.py", line 57, in discover
    plugin_descriptors = plugin_provider.discover(discovery_data)
  File "/opt/ros/noetic/lib/python3.9/site-packages/qt_gui/recursive_plugin_provider.py", line 53, in discover
    plugin_descriptors = self._plugin_provider.discover(discovery_data)
  File "/opt/ros/noetic/lib/python3.9/site-packages/rqt_gui/ros_plugin_provider.py", line 68, in discover
    plugin_descriptors += self._parse_plugin_xml(package_name, plugin_xml)
  File "/opt/ros/noetic/lib/python3.9/site-packages/rqt_gui/ros_plugin_provider.py", line 121, in _parse_plugin_xml
    for library_el in root.getiterator('library'):
AttributeError: 'ElementTree' object has no attribute 'getiterator'

 provider "<class 'qt_gui.recursive_plugin_provider.RecursivePluginProvider'>":
Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3.9/site-packages/qt_gui/composite_plugin_provider.py", line 57, in discover
    plugin_descriptors = plugin_provider.discover(discovery_data)
  File "/opt/ros/noetic/lib/python3.9/site-packages/qt_gui/recursive_plugin_provider.py", line 53, in discover
    plugin_descriptors = self._plugin_provider.discover(discovery_data)
  File "/opt/ros/noetic/lib/python3.9/site-packages/rqt_gui/ros_plugin_provider.py", line 68, in discover
    plugin_descriptors += self._parse_plugin_xml(package_name, plugin_xml)
  File "/opt/ros/noetic/lib/python3.9/site-packages/rqt_gui/ros_plugin_provider.py", line 121, in _parse_plugin_xml
    for library_el in root.getiterator('library'):
AttributeError: 'ElementTree' object has no attribute 'getiterator'

 provider "<class 'qt_gui.recursive_plugin_provider.RecursivePluginProvider'>":
Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3.9/site-packages/qt_gui/composite_plugin_provider.py", line 57, in discover
    plugin_descriptors = plugin_provider.discover(discovery_data)
  File "/opt/ros/noetic/lib/python3.9/site-packages/qt_gui/recursive_plugin_provider.py", line 53, in discover
    plugin_descriptors = self._plugin_provider.discover(discovery_data)
  File "/opt/ros/noetic/lib/python3.9/site-packages/rqt_gui/ros_plugin_provider.py", line 68, in discover
    plugin_descriptors += self._parse_plugin_xml(package_name, plugin_xml)
  File "/opt/ros/noetic/lib/python3.9/site-packages/rqt_gui/ros_plugin_provider.py", line 121, in _parse_plugin_xml
    for library_el in root.getiterator('library'):
AttributeError: 'ElementTree' object has no attribute 'getiterator'
```

**Comment by cubercsl** [comment]
Meet same issue too. Just change `getiterator` to `iter` may fix this.

Also affacted https://github.com/ros-noetic-arch/ros-noetic-rqt/issues/1

**Comment by hoser21** [comment]
> Meet same issue too. Just change `getiterator` to `iter` may fix this.
> 
> Also affacted [ros-noetic-arch/ros-noetic-rqt#1](https://github.com/ros-noetic-arch/ros-noetic-rqt/issues/1)

This fixes the issue for me.

**Comment by sven-herrmann** [comment]
Was merged in #241 closing
